kitodo-mediaserver-ui/src/main/java/org/kitodo/mediaserver/ui/config/WebMvcConfiguration.java

```diff
@@ -44,7 +42,7 @@
  * App MVC Configuration.
  */
 @Configuration
-@ComponentScan(basePackages = {"org.kitodo.mediaserver.ui", "org.kitodo.mediaserver.core.services"})
+@ComponentScan(basePackages = {"org.kitodo.mediaserver.ui", "org.kitodo.mediaserver.core"})
 @Import({FileserverProperties.class, MetsProperties.class})
```

Comment:
I'm not sure this is necessary any more, since you do a component scan of core

Reply:
You're right. It works without this line now.